packages/data-contract/src/constants.js

```diff
@@ -0,0 +1,28 @@
+const schema = {
+    "labler": {
```

Comment:
Document type should describe a type of documents that will be stored under that name. Since we are going to deploy PE data contract, it will contain different document types for different purposes. In this case, I suggest something like `dataContracts`